products/fedora.md

```diff
@@ -10,11 +10,10 @@ sortReleasesBy: 'releaseCycle'
 changelogTemplate: https://fedoraproject.org/wiki/Releases/__LATEST__
 category: os
 releases:
-  # feature release 35 expected on 2021-11-02 -> calculate EoL of 33
-  #- releaseCycle: "35"
-  #  release: 2021-11-02
-  #  latest: "35"
-  #  eol: false
+  - releaseCycle: "35"
+    release: 2021-11-02
+    latest: "35"
+    eol: false
```

Comment:
```suggestion
    eol: 2022-12-07
```
https://fedorapeople.org/groups/schedule/f-35/f-35-all-tasks.html

Reply:
Thanks for this, I didn't saw that they have already a date in the schedule.